restapi/stories.go

```diff
@@ -332,12 +351,11 @@ func (api api) uploadImageTos3(file multipart.File, size int64, name string) (st
 	if err != nil {
 		return "", fmt.Errorf("Could not upload to S3")
 	}
-	return fmt.Sprintf("https://%s.s3.us-west-000.backblazeb2.com/%s", os.Getenv("BUCKET_NAME"), name), nil
+	return fmt.Sprintf("https://%s.s3.us-west-000.backblazeb2.com/%s?versionId=%s", os.Getenv("BUCKET_NAME"), name, *res.VersionId), nil
```

Comment:
remove this please

Reply:
Remove what? VersionId is needed here.

Reply:
oh my bad i thought you were logging something here nvm